packages/web-components/src/components/va-maintenance-banner/va-maintenance-banner.scss

```diff
@@ -0,0 +1,86 @@
+@forward 'settings';
+@use 'uswds-helpers/src/styles/usa-sr-only';
+@use 'usa-button/src/styles/usa-button';
+@use 'usa-icon/src/styles/usa-icon';
+
+@import '../../global/formation_overrides';
```

Comment:
Are we adding a USWDS v3 variation at the same time? These modules would be specific to that but I don't see a `uswds` prop.

```
@forward 'settings';
@use 'uswds-helpers/src/styles/usa-sr-only';
@use 'usa-button/src/styles/usa-button';
@use 'usa-icon/src/styles/usa-icon';

@import '../../global/formation_overrides';
```

Reply:
Removed unused CSS imports. Not USWDS yet